airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/source.py

```diff
@@ -254,6 +254,20 @@ def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[
         params["q"] = self.search_param
         return params
 
+    def read_records(
+        self, stream_state: Mapping[str, Any] = None, stream_slice: Optional[Mapping[str, Any]] = None, **kwargs
+    ) -> Iterable[Mapping[str, Any]]:
+        stream_state = stream_state or {}
+        parent_stream = self.parent_stream(config=self.config)
+        for record in parent_stream.read_records(**kwargs):
+
+            if record.get('reference', '').startswith('urn:li:person'):
+                self.logger.warn(f'Skip {record.get("name")} account because it has reference to person {record.get("reference")} instead of organization')
```

Comment:
nit: 
Skipping stream <stream_name> for account <account>, ORGANIZATION permissions required, but referenced to PERSON.